lib/workers/repository/process/lookup/rollback.ts

```diff
@@ -39,7 +39,15 @@ export function getRollbackUpdate(
     'Versions found before rolling back'
   );
   lessThanVersions.sort((a, b) => version.sortVersions(a.version, b.version));
-  const newVersion = lessThanVersions.pop()?.version;
+  let newVersion;
+  if (version.isStable(currentValue)) {
```

Comment:
```suggestion
  if (version.isStable(currentValue!)) {
```

Reply:
![image](https://user-images.githubusercontent.com/98313426/171828267-562fd5e7-24a1-4dc3-8c65-868106328b47.png)

Reply:
Your way of solving was fine too. It's acceptable to put an eslint ignore for this particular case if necessary in future